std/array.d

```diff
@@ -2651,15 +2651,19 @@ body
 
 /**
 Implements an output range that appends data to an array. This is
-recommended over $(D a ~= data) when appending many elements because it is more
-efficient.
+recommended over $(D array ~= data) when appending many elements because it is more
+efficient. `Appender` maintains its own array metadata locally, so it can avoid
+global locking for each append where $(LREF capacity) is non-zero.
+See_Also: $(LREF appender)
  */
 struct Appender(A)
 if (isDynamicArray!A)
 {
     import core.memory : GC;
 
-    private alias T = ElementEncodingType!A;
+    ///
+    package alias T = ElementEncodingType!A;
```

Comment:
Instead of documenting this, it would make more sense to replace it in the signitures.